shared/src/types/admin/Recipient.ts

```diff
@@ -10,6 +10,16 @@ export enum RecipientProgramStatus {
 	Former = 'former',
 }
 
+// TODO change to using a standard like ISO 639-3.
+export enum RecipientMainLanguage {
+	Krio = 'krio',
```

Comment:
Wouldn't it make sense to already use the respective values, i.e. `kri` for Krio, `men` for Mende, etc.?

See: https://iso639-3.sil.org/sites/iso639-3/files/downloads/iso-639-3.tab

Reply:
Ah ok, I see the problem. We already have entries in Firestore... I guess we need a script that converts the current entries...

Reply:
Currently all 'mainLanguageProperty' (First Language) values are set to Krio (best guess) or not set at all. As there were not yet any information in a local languages, these fields were not yet maintained or kept up to date. So they can be easily erased or all better overwritten with `kri`.

If it helps @andras here the ISO codes for each language: :

- Krio = _kri_
- Mende = _men_
- Temne = _tem_
- Limba = _lia_
- plus English _en_ and _other_

Reply:
Ah perfect! In that case, I will change it to those and change the values in the DB. 👍